### PR TITLE
Remove deprecated module

### PR DESCRIPTION
**Pull request by Ruchip16**
##### SUMMARY

Removes deprecated ` iosxr_logging ` legacy module in favour of `iosxr_logging_global` 

##### ISSUE TYPE

-  Remove deprecated module

##### COMPONENT NAME

`iosxr_logging`

### CODE REVIEW

**Comment by ashwini-mhatre** [comment]
please remove unit test case file also if any

**Comment by Ruchip16** [comment]
> please remove unit test case file also if any

yes, there is no unit test-case file for `iosxr_logging`

File: changelogs/fragments/remove_deprecated.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (NilashishC) [inline changelogs/fragments/remove_deprecated.yml:1]: Please refer to https://github.com/ansible-collections/cisco.nxos/pull/836/files#diff-044354a2e5989396b9168d5dde63bd5196447bfcf661637e7433555b532c50c6R1-R8 for changelog entry.

**Review by NilashishC — CHANGES_REQUESTED** [review]
The meta/runtime.yml entry should also be removed.

**Review by KB-perByte — APPROVED** [review]

**Review by NilashishC — APPROVED** [review]

**Pull request merged**